### PR TITLE
Disable failing EventLogSession.GetProviderNames test

### PR DESCRIPTION
**Pull request by sbomer**
See https://github.com/dotnet/runtime/issues/64153

### CODE REVIEW

**Comment by carlossanlop** [comment]
@tarekgh any chance you could help take a look at this one?

**Comment by tarekgh** [comment]
@carlossanlop I couldn't repro this locally. Can you get me a dump from any run that was failing?

**Comment by tarekgh** [comment]
@sbomer I am seeing this is failing on x86 too. Why are we disabling this test on x64 only?

File: src/libraries/System.Diagnostics.EventLog/tests/System/Diagnostics/Reader/ProviderMetadataTests.cs [hunk 1]
```diff
@@ -90,6 +91,7 @@ public void ProviderNameTests(bool noProviderName)
         }
 
         [ConditionalFact(typeof(Helpers), nameof(Helpers.SupportsEventLogs))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/64153", typeof(PlatformDetection), nameof(PlatformDetection.IsWindows), nameof(PlatformDetection.IsX64Process))]
```
Comment (tarekgh) [inline src/libraries/System.Diagnostics.EventLog/tests/System/Diagnostics/Reader/ProviderMetadataTests.cs:94]: >nameof(PlatformDetection.IsWindows), nameof(PlatformDetection.IsX64Process)) [](http://example.com/codeflow?start=98&length=76)

nit: you can combine these 2 conditions in PlatformDetection. Something like `IsX64ProcessOnWindows`

Comment (sbomer) [reply src/libraries/System.Diagnostics.EventLog/tests/System/Diagnostics/Reader/ProviderMetadataTests.cs]: Done!

**Comment by sbomer** [comment]
> @sbomer I am seeing this is failing on x86 too. Why are we disabling this test on x64 only?

IIRC originally I saw it failing only on x64 and I wasn't sure if it was a recurring failure. Now that we are seeing it more often and on x86 I updated the PR to disable it on both architectures. Thanks!

**Review by tarekgh — APPROVED** [review]
Thanks @sbomer

**Pull request merged**